.github/workflows/gradle-publish.yml

```diff
@@ -8,7 +8,9 @@
 name: Gradle Package
 
 on:
-  workflow_dispatch
+  workflow_dispatch:
+  push:
+    tags: 'v*'
```

Comment:
Wouldn't the following have been sufficient after the _workflow_dispatch_? (@omarsilva1 @efiege)
```
   release:
     types: [published]
```

Reply:
Yeah this should also work! :)

Reply:
I like this solution. Artifacts are published when releasing on github. Please reapprove the PR.